movo_robot/movo_upstart/templates/job-start-movo.em

```diff
@@ -118,7 +118,7 @@ fi
 
 temp=1
 cnts=0
-while [ $cnts -lt 10 ] && [ $temp -ne 0 ]; do
+while [ $cnts -lt 100 ] && [ $temp -ne 0 ]; do
```

Comment:
why change this value?

Reply:
This value corresponds to the number of tries the robot will ping its own arms. Sometimes, I was getting errors telling me that Movo couldn't find right arm or left arm. By increasing the number of pings, it lets enough time for Movo to correctly ping its arms.